crates/astria-composer/src/searcher/executor.rs

```diff
@@ -407,11 +466,420 @@ impl Future for SubmitFut {
 }
 
 fn sha256(data: &[u8]) -> [u8; 32] {
-    use sha2::{
-        Digest as _,
-        Sha256,
+    use sha2::Sha256;
+    Sha256::digest(data)
+}
+
+#[cfg(test)]
+mod tests {
```

Comment:
Please move `executor.rs -> executor/mod.rs` and this module into its own `executor/tests.rs`